### ISSUE TITLE
[fvt]2.12.2 xcatprobe osdeploy -n could not monitor the node deploy process if message's log : dhcp[process id]

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1717]
env:ubuntu16.04.1
build:
root@c910f04x21v02:~# lsxcatd  -v
Version 2.12.2 (git commit 9298b5ff211500e22398abd7c35b3fe28038feff, built Tue Aug 16 09:12:17 EDT 2016)
root@c910f04x21v02:~# 
How to reproduce:
``` 
xcatprobe osdeploy -n c910f04x21v04
 

All pre_defined nodes are valid [ OK ]
The installation network interface is ens3 [INFO]
-------------------------------------------------------------
Start to capture every message during os provision process......
-------------------------------------------------------------

[c910f04x21v04] Node status is changed to powering-on
[c910f04x21v04] Node status is changed to installing
[c910f04x21v04] xcat Wed Aug 17 02:26:00 EDT 2016 Running postscript: syslog
[c910f04x21v04] xcat Wed Aug 17 02:26:00 EDT 2016 postscript syslog return with 0
[c910f04x21v04] xcat Wed Aug 17 02:26:01 EDT 2016 Running postscript: remoteshell
[c910f04x21v04] xcat Wed Aug 17 02:26:04 EDT 2016 postscript remoteshell return with 0
[c910f04x21v04] xcat Wed Aug 17 02:26:04 EDT 2016 Running postscript: syncfiles
[c910f04x21v04] xcat Wed Aug 17 02:26:04 EDT 2016 postscript syncfiles return with 0
[c910f04x21v04] Node status is changed to booting
[c910f04x21v04] xcat running /xcatpost/mypostscript.post
[c910f04x21v04] xcatpostinit1[1834]: Wed Aug 17 02:26:39 EDT 2016 Running postscript: otherpkgs
[c910f04x21v04] xcatpostinit1[1834]: Wed Aug 17 02:26:39 EDT 2016 Running postscript: otherpkgs
[c910f04x21v04] xcat Wed Aug 17 02:26:39 EDT 2016 Running postscript: otherpkgs
[c910f04x21v04] xcat: Running otherpkgs
[c910f04x21v04] xcatpostinit1[1834]: ./otherpkgs: no extra rpms to install
[c910f04x21v04] xcatpostinit1[1834]: ./otherpkgs: no extra rpms to install
[c910f04x21v04] xcatpostinit1[1834]: Postscript: otherpkgs exited with code 0
[c910f04x21v04] xcat Wed Aug 17 02:26:39 EDT 2016 postscript otherpkgs return with 0
[c910f04x21v04] xcat: ready
[c910f04x21v04] Node status is changed to booted
All nodes need to monitor have finished os provision process [ OK ]

=============================================================
= The summary of os provision:
=============================================================

[c910f04x21v04] reboot completed, without deployment process [FAIL]
 tail -f /var/log/message:
root@c910f04x21v02:~# tail -f /var/log/messages 
Aug 17 03:45:27 c910f04x21v02 dhcpd[32306]: DHCPDISCOVER from 42:63:0a:03:11:12 via ens3: network ens3: no free leases
Aug 17 03:45:28 c910f04x21v02 dhcpd[32306]: DHCPDISCOVER from 16:3f:48:47:a3:03 via ens3: network ens3: no free leases
Aug 17 03:45:30 c910f04x21v02 dhcpd[32306]: DHCPREQUEST for 10.2.3.15 from 16:3f:4b:73:e4:03 via ens3: unknown lease 10.2.3.15.
Aug 17 03:45:32 c910f04x21v02 dhcpd[32306]: DHCPDISCOVER from da:08:4c:8c:d6:03 via ens3: network ens3: no free leases
Aug 17 03:45:33 c910f04x21v02 dhcpd[32306]: DHCPDISCOVER from 4a:c8:f9:15:61:03 via ens3: network ens3: no free leases
Aug 17 03:45:33 c910f04x21v02 dhcpd[32306]: DHCPREQUEST for 10.2.8.10 (10.2.8.8) from 4a:c8:f9:15:61:03 via ens3: unknown lease 10.
----------------------------> In fact the node provisioned successfully

```

**Comment by hu-weihua** [comment]
@junxiawang ,  I have fixed this bug by pull request  #1726.Could you help to verify it ? thanks

**Comment by junxiawang** [comment]
verified
